shpool/tests/data/motd_pager_env_test.toml.tmpl

```diff
@@ -0,0 +1,11 @@
+norc = true
+noecho = true
+shell = "/bin/bash"
+session_restore_mode = "simple"
+
+[motd.pager]
+bin = "MOTD_ENV_TEST_SCRIPT"
+
+[env]
+PS1 = "prompt> "
```

Comment:
very clear.


any chance we can also add comments at the top of this file to say more on the intent? (although it can help reading, it is okay if we can't, since the test that calls it is the source of truth)